plugins/filter_aws/aws.c

```diff
@@ -95,11 +124,19 @@ static int cb_aws_init(struct flb_filter_instance *f_ins,
     /* v1 or v2 instance metadata */
     ctx->use_v2 = use_v2;
 
-    /* enabled fields */
+    /* get the enabled fields from the config file*/
+    ctx->instance_type_include = set_enabled_fields("ec2_instance_type", f_ins, &new_keys);
+    ctx->private_ip_include = set_enabled_fields("private_ip", f_ins, &new_keys);
+    ctx->vpc_id_include = set_enabled_fields("vpc_id", f_ins, &new_keys);
+    ctx->ami_id_include = set_enabled_fields("ami_id", f_ins, &new_keys);
+    ctx->account_id_include = set_enabled_fields("account_id", f_ins, &new_keys);
+    ctx->hostname_include = set_enabled_fields("hostname", f_ins, &new_keys);
+
+    /* default enabled fields */
     ctx->instance_id_include = FLB_TRUE;
     ctx->availability_zone_include = FLB_TRUE;
```

Comment:
The user should be able to disable these default fields. Is that possible with the current code?

Reply:
I was thinking we might plan to keep these two as default. 
But yeah it both make sense to me.
Changed to allow customer disable it.